### PR TITLE
Cargo pack Stuff

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Removes a few cruftpacks
Alters descs on a few other packs
Adds like one pack
moves fishing into its own category

## Why It's Good For The Game
Cargo continues to become more usuable

## Changelog

:cl:
add: Some cargo packs have new descriptions
code: cargo is slightly more organized. again
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/exploration.dm [hunk 1]
```diff
@@ -48,48 +32,21 @@
...
-		/obj/item/clothing/glasses/cheapsuns,
-		/obj/item/clothing/glasses/cheapsuns,
-	)
-	crate_name = "Beach Kit"
 
 /*
 		General Exploration Gear
 */
 
+/datum/supply_pack/exploration/lavaproof_ords
```
Comment (Sadhorizon) [inline code/modules/cargo/packs/exploration.dm]: ords

**Comment by Anticept** [comment]
I didn't know the jungle kit came with lanterns! Want!

**Comment by Orchidthederg** [comment]
i'm afraid this is peak...

**Review by Sun-Soaked — APPROVED** [review]
RIP mowerpack

**Pull request merged**